### ISSUE TITLE
Function-per-file export

### ISSUE THREAD

**Issue by omgovich** [swiftcarrot/color-fns#151]
Is is possible to implement function-per-file style exports like in [`date-fns`](https://date-fns.org)?
That allows avoiding tree-shaking issues and import only the methods you need.

```js
import hex2hsl from '@swiftcarrot/color-fns/hex2hsl'
```

**Comment by wangzuo** [comment]
This package should work with tree-shaking bundling.

**Comment by omgovich** [comment]
Yeah, right. But there is a case that function-per-file style could solve. For example:
I'm working on my library (it's a color-picker for React) that has `swiftcarrot/color-fns` as a dependency.
If someone will install my library and won't turn tree shaking in his project's bundler, the whole `swiftcarrot/color-fns` will be imported to his bundle.

Chance that tree-shaking is will be disabled in the user's config is pretty high, so I would like to avoid importing extra code by importing just functions I need by my library's code.

**Comment by wangzuo** [comment]
Yes, it does make sense. The `cssColor` method takes up most of the size of this package, which is not used in most cases. We may need a new build setup to support entrypoint per method.
